src/StartScreen.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { StyleSheet, Text, View, Image, TouchableHighlight } from 'react-native';
+
+export const StartScreen = () => {
+    const goRead = () => {}
+
+    return (
+        <View style={styles.startScreen}>
+            <Image
+                style={styles.logo}
+                source={require('../assets/wordmark.png')}
+            />
+            <TouchableHighlight
+                onPress={goRead}
+                style={styles.highlight}
+            >
+                <View style={styles.buttonContainer}>
+                    <Image
+                        style={styles.button}
+                        source={require('../assets/Open.png')}
+                    />
+                    <Text style={styles.text}>Choose BloomPUB book on this computer</Text>
```

Comment:
Maybe "device" or "phone" instead of "computer" now?

Reply:
I went with 'device' for now, since they could be using an ipad or something.